marlowe-symbolic/src/Language/Marlowe/Analysis/FSSemantics.hs

```diff
@@ -185,8 +185,8 @@ symEvalVal SlotIntervalStart symState = lowSlot symState
 symEvalVal SlotIntervalEnd symState = highSlot symState
 symEvalVal (UseValue valId) symState =
   M.findWithDefault (literal 0) valId (symBoundValues symState)
-symEvalVal (Cond cond v1 v2) symState = ite (symEvalObs cond symState) 
-                                            (symEvalVal v1 symState) 
+symEvalVal (Cond cond v1 v2) symState = ite (symEvalObs cond symState)
+                                            (symEvalVal v1 symState)
                                             (symEvalVal v2 symState)
 
 -- Symbolic version evalObservation
```

Comment:
Should this really be in here?  It's just a whitespace difference.  Was that done automatically?

Reply:
Yup, I ran stylish.